design/overview.js

```diff
@@ -0,0 +1,56 @@
+//Set global used to evade unnecesary builds
+var prev_column_count = -1;
```

Comment:
Ah, I was wondering if we did this correctly. Thanks for checking wether builds aren't necessary :clap: